pkgs/by-name/li/libtorrent-rasterbar-1_2_x/package.nix

```diff
@@ -17,17 +30,21 @@ in stdenv.mkDerivation {
     owner = "arvidn";
     repo = "libtorrent";
     rev = "v${version}";
-    hash = "sha256-KxyJmG7PdOjGPe18Dd3nzKI5X7B0MovWN8vq7llFFRc=";
+    hash = "sha256-HkpaOCBL+0Kc7M9DmnW2dUGC+b60a7n5n3i1SyRfkb4=";
   };
 
   enableParallelBuilding = true;
 
-  nativeBuildInputs = [ automake autoconf libtool pkg-config ];
+  nativeBuildInputs = [ automake autoconf boost-build libtool pkg-config python311.pkgs.setuptools ];
 
-  buildInputs = [ boostPython openssl zlib python311 libiconv ncurses ]
+  buildInputs = [ boostPython opensslStatic zlib python311 libiconv ncurses ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [ darwin.apple_sdk.frameworks.SystemConfiguration ];
 
-  preConfigure = "./autotool.sh";
+  preConfigure = ''
+    ./autotool.sh
```

Comment:
You may want to look into `autoreconfHook` (you can use it by adding it to nativeBuildInputs)

Reply:
I've amended the build to use `autoreconfHook`.